src/main/java/org/folio/circulation/domain/Location.java

```diff
@@ -0,0 +1,105 @@
+package org.folio.circulation.domain;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import org.folio.circulation.support.JsonPropertyFetcher;
+
+import io.vertx.core.json.JsonObject;
+
+public class Location {
+
+  private final JsonObject representation;
+  private final JsonObject libraryRepresentation;
+  private final JsonObject campusRepresentation;
+  private final JsonObject institutionRepresentation;
+
+  public Location(JsonObject representation, JsonObject libraryRepresentation, JsonObject campusRepresentation, JsonObject institutionRepresentation) {
```

Comment:
Please wrap these constructor parameters to the intended 80 character line length limit

Reply:
done